experimental/packages/opentelemetry-instrumentation-grpc/test/grpc-js.test.ts

```diff
@@ -24,5 +25,72 @@ instrumentation.disable();
 import '@grpc/grpc-js';
 
 describe('#grpc-js', () => {
-  runTests(instrumentation, 'grpc', 12346);
+  runTestsWithSemconvStabilityLevels(instrumentation, 'grpc', 12346);
+});
+
+describe('grpc-js semconv stability', () => {
```

Comment:
Personally I think you could drop this test after switching to the `httpSemconvStabilityFromStr` from `@opentelemetry/instrumentation`.

Reply:
Agreed!